roles/ceph-infra/handlers/main.yml

```diff
@@ -3,4 +3,22 @@
   service:
     name: firewalld
     state: restarted
-    enabled: yes
\ No newline at end of file
+    enabled: yes
+- name: disable ntpd
+  failed_when: false
```

Comment:
@leseb @guits Instead of checking if the service is present, I am letting the task pass if it fails (when the service is absent). I preferred this since it has the same effect and it would consume lesser lines. Is that OK?

Reply:
I think it's fine, yes.